Qt.py

```diff
@@ -43,6 +43,23 @@ def _pyqt5():
 
 
 def _pyqt4():
+    # Attempt to set sip API v2 (must be done prior to importing PyQt4)
+    import sip
+    try:
+        sip.setapi("QString", 2)
+        sip.setapi("QVariant", 2)
+        sip.setapi("QDate", 2)
+        sip.setapi("QDateTime", 2)
+        sip.setapi("QTextStream", 2)
+        sip.setapi("QTime", 2)
+        sip.setapi("QUrl", 2)
+    except AttributeError as error:
```

Comment:
This works, but leaves the impression that someone down the line will catch this error and do something with it.

This is more clear I think.

```python
except AttributeError:
  # PyQt4 < v4.6
  raise ImportError

except ValueError:
  # API version already set to v1
  raise ImportError
```